packages/x-components/src/x-modules/next-queries/components/next-queries.vue

```diff
@@ -134,17 +192,36 @@ the `emitNextQuerySelected` function when the next query is selected. In the exa
 of using the default `button` tag for a next query, an icon is added, and the text of the next query
 is wrapped in a `span`
 
-```vue
-<NextQueries>
-  <template #suggestion="{suggestion}">
-    <NextQuery :suggestion="suggestion" class="x-next-queries__suggestion">
-      <template #default="{suggestion}">
-        <img src="./next-query-icon.svg" class="x-next-query__icon"/>
-        <span class="x-next-query__query">{{ suggestion.query }}</span>
+```vue live
+<template>
+  <div>
+    <SearchInput />
+    <NextQueries>
+      <template #suggestion="{ suggestion }">
+        <NextQuery :suggestion="suggestion" class="x-next-queries__suggestion">
+          <template #default="{ suggestion }">
+            <img src="/assets/icons/bulb.svg" class="x-next-query__icon" />
```

Comment:
Change this icon for the BulbOn, the one from the previous comment.

Reply:
I am trying to add more specific icons, matching this component, i added `Nq1Icon`.